tests/ocf-meta-inf_manifest/EPUB/package.opf

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-05-28</dc:date>
+    <dc:description>An ancillary manifest file, containing an extra spine item, is present in the meta-inf directory; this extra item must be ignored by the reading system.</dc:description>
```

Comment:
I think this isn't the only thing happening in this test, is it? I see content.xml *and* manifest.xml.

Reply:
Since the manifest.xml is the actual test target and content.xml is used to support or verify the test I am going to leave it for now. Please feel free to let me know if you think better making description change I can send a separate PR later.